dataprofiler/profilers/profile_builder.py

```diff
@@ -780,6 +780,76 @@ def update_profile(self, data, sample_size=None, min_true_samples=None):
 
         self._update_profile_from_chunk(data, sample_size, min_true_samples)
 
+    def _remove_data_labelers(self):
+        """
+        Helper method for removing all data labelers before saving to disk.
+
+        :return: data_labeler use for unstructured labeling
+        :rtype: DataLabeler
+        """
+        data_labeler = None
+
+        # Delete data labeler for profiler
+        # TODO: fix to correct options when unstructured options avail
+        use_data_labeler = True
+        if self.options and isinstance(self.options, ProfilerOptions):
+            data_labeler_options = self.options.structured_options.data_labeler
+            use_data_labeler = data_labeler_options.is_enabled
+        if use_data_labeler \
+                and data_labeler_options.data_labeler_object is not None:
+            data_labeler = data_labeler_options.data_labeler_object
+            data_labeler_options.data_labeler_object = None
+
+        # remove data labelers
+        # TODO: fix to correct options when unstructured options avail
+        if use_data_labeler:
+            if data_labeler is None:
+                data_labeler = \
+                    self._profile._profiles['data_labeler'].data_labeler
+            self._profile._profiles['data_labeler'].data_labeler = None
+
+        return data_labeler
+
+    def _restore_data_labelers(self, data_labeler=None):
+        """
+        Helper method for restoring all data labelers after saving to or
+        loading from disk.
+
+        :param data_labeler: unstructured data_labeler
+        :type data_labeler: DataLabeler
+        """
+        # Restore data labeler for profiler
+        # TODO: fix to correct options when unstructured options avail
+        use_data_labeler = True
+        data_labeler_options = None
+        if self.options and isinstance(self.options, ProfilerOptions):
+            data_labeler_options = self.options.structured_options.data_labeler
+            use_data_labeler = data_labeler_options.is_enabled
+
+        if use_data_labeler:
+            try:
+                if data_labeler is None and data_labeler_options is None:
+                    data_labeler = DataLabeler(
+                        labeler_type='unstructured',
+                        dirpath=None,
+                        load_options=None)
+                elif data_labeler is None:
+                    data_labeler = DataLabeler(
+                        labeler_type='unstructured',
+                        dirpath=data_labeler_options.data_labeler_dirpath,
+                        load_options=None)
+                self.options.set(
+                    {'data_labeler.data_labeler_object': data_labeler})
+
+            except Exception as e:
+                utils.warn_on_profile('data_labeler', e)
+                self.options.set({'data_labeler.is_enabled': False})
+
+        # Restore data labelers for all columns
```

Comment:
Doesn't "all columns" not really make sense here? Is this just a comment from the structured version of this function?

Reply:
Yes, will fix.